editor/editor_properties.cpp

```diff
@@ -2849,10 +2892,17 @@ EditorProperty *EditorInspectorDefaultPlugin::get_editor_for_property(Object *p_
 						if (slice == "or_lesser") {
 							lesser = true;
 						}
+						if (slice.begins_with("suffix:")) {
+							suffix = " " + slice.replace_first("suffix:", "").strip_edges();
+						}
 					}
 				}
 
-				editor->setup(min, max, step, hide_slider, exp_range, greater, lesser);
+				if (angle && suffix == String()) {
```

Comment:
```suggestion
				if (angle && suffix.is_empty()) {
```

Reply:
Sorry, I like my syntax more.

Reply:
@reduz `.is_empty()` is marginally faster, see the benchmarks in #43181.

Reply:
I don't care, it's not critical in this context.